### PR TITLE
added build instructions

### PR DESCRIPTION
**Pull request by dylanmc**
Yes, it's silly, but I thought non-poetry folks (like myself) could use the help with installation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,14 @@ Install from pip:
 pip install aga
 ```
 
-or with the python dependency manager of your choice (I like [poetry](https://github.com/python-poetry/poetry)).
+or with the python dependency manager of your choice (I like [poetry](https://github.com/python-poetry/poetry)), for example:
+
+```bash
+pip install poetry
```
Comment (nihilistkitten) [inline README.md:35]: Can we put the [official](https://python-poetry.org/docs/#installation) installation method? Just change this line to

```bash
curl -sSL https://install.python-poetry.org | python3 -
```

Comment (nihilistkitten) [reply README.md:35]: (Installing python libraries globally via `pip` is almost always a bad idea.)

**Comment by nihilistkitten** [comment]
Good idea, just suggested one change.

**Comment by dylanmc** [comment]
> Good idea, just suggested one change.

Makes sense!

File: README.md (hunk 1 above)
Comment (dylanmc) [reply README.md]: I have tried to fix my PR with your suggestion, but I'm getting mixed messages from github. Fingers crossed.

**Pull request merged**